demo/src/main/java/iamutkarshtiwari/github/io/imageeditorsample/MainActivity.java

```diff
@@ -252,8 +254,9 @@ private void loadImage(String imagePath) {
                 .doFinally(() -> loadingDialog.dismiss())
                 .subscribe(
                         this::setMainBitmap,
-                        e -> Toast.makeText(
-                                this, R.string.iamutkarshtiwari_github_io_ananas_load_error, Toast.LENGTH_SHORT).show()
+                        e -> { e.printStackTrace();
```

Comment:
Awesome!

Reply:
this was help full because we know what is the reason for error. i needed it for fix the image capture error fix so added that line.